distributed/deploy/tests/test_ssh.py

```diff
@@ -19,6 +19,7 @@ def test_ssh_hosts_empty_list():
         SSHCluster(hosts=[])
 
 
+@pytest.mark.ssh
```

Comment:
I wonder if we can mark the entire file with `ssh` somehow?

Reply:
I wondered that too but found nothing on the pytest docs

Reply:
It looks like there's a special `pytestmark` variable we can set at the module-level https://docs.pytest.org/en/stable/example/markers.html#marking-whole-classes-or-modules

Reply:
fixed